java/pom.xml

```diff
@@ -233,36 +235,36 @@
                 </executions>
             </plugin>
 
-<!--            <plugin>-->
-<!--                <groupId>au.com.dius.pact.provider</groupId>-->
-<!--                <artifactId>maven</artifactId>-->
-<!--                <version>4.2.7</version>-->
-<!--                <configuration>-->
-<!--                    <pactBrokerUrl>${pact.broker.url}</pactBrokerUrl>-->
-<!--                    <pactBrokerToken>${pact.broker.token}</pactBrokerToken>-->
-<!--                    <systemPropertyVariables>-->
-<!--                        <pact.verifier.publishResults>true</pact.verifier.publishResults>-->
-<!--                    </systemPropertyVariables>-->
-
-<!--                    <serviceProviders>-->
-<!--                        <serviceProvider>-->
-<!--                            <name>${pact.api.name}</name>-->
-<!--                            <protocol>${pact.api.protocol}</protocol>-->
-<!--                            <host>${pact.api.host}</host>-->
-<!--                            <port>${pact.api.port}</port>-->
-<!--                        </serviceProvider>-->
-<!--                    </serviceProviders>-->
-
-<!--                </configuration>-->
-<!--                <executions>-->
-<!--                    <execution>-->
-<!--                        <phase>integration-test</phase>-->
-<!--                        <goals>-->
-<!--                            <goal>verify</goal>-->
-<!--                        </goals>-->
-<!--                    </execution>-->
-<!--                </executions>-->
-<!--            </plugin>-->
+            <!--            <plugin>-->
```

Comment:
revert this?

Reply:
I'm not sure what to do with the PACT configuration. We need to agree on this